src/foam/u2/borders/NullBorder.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.borders',
+  name: 'NullBorder',
+  extends: 'foam.u2.Element',
+
+  documentation: `
+    An unstyled border. Intended for use as a default value for
+    border properties.
+  `,
+
+  methods: [
+    function initE() {
```

Comment:
If you don't provide this initE() method does the standard Element just work?

Reply:
I just tried and it doesn't.